src/test/data/JsonSerializableModuleBookTest/invalidTaskModuleBook.json

```diff
@@ -3,6 +3,7 @@
     "name": "Hans Muster",
     "deadline": "9482424",
     "module": "invalid@module!3e",
-    "description": "4th street"
+    "description": "4th street",
+    "doneStatus": "false"
   } ]
 }
```

Comment:
Here the value of property isn't change to correct format of MB3.5.

Reply:
To clarify, is the deadline here supposed to change?